maxim/tests/test_maxim_core_simple.py

```
@@ -0,0 +1,246 @@
+    def test_maxim_default_cache_creation(self):
+        """Test that Maxim creates default cache when none provided."""
+        maxim = Maxim({"api_key": "test-key"})
+
+        cache = getattr(maxim, "_Maxim__cache")
+        self.assertIsInstance(cache, MaximInMemoryCache)
+
+    def test_cleanup_method_stops_running(self):
+        """Test that cleanup method properly sets is_running to False."""
+        maxim = Maxim({"api_key": "test-key"})
+
+        self.assertTrue(maxim.is_running)
```

Ruff (0.11.9)

Use a regular `assert` instead of unittest-style `assertTrue`

Replace `assertTrue(...)` with `assert ...`

(PT009)